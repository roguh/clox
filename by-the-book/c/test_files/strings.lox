"abcdef" '123' "123456\""
