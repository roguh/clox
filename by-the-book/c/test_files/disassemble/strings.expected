== compileAndPrint ==
0000    1 OP_CONSTANT         0 'abcdef'
0002    | OP_POP
0003    | OP_CONSTANT         1 '123'
0005    | OP_POP
0006    | OP_CONSTANT         2 '123456\"'
0008    | OP_POP
0009    3 OP_CONSTANT         3 'multi
line
string!'
0011    | OP_POP
0012    4 OP_CONSTANT         4 'after multiline?'
0014    | OP_POP
0015    5 OP_CONSTANT         5 'abc'
0017    | OP_POP
0018    | OP_RETURN
